cartoframes/assets/templates/viz/widgets.js.j2

```diff
@@ -20,10 +20,14 @@ function renderBridge(bridge, widget) {
       break;
     case 'animation':
       const options = {
-        propertyName: widget.prop
+        propertyName: widget.prop,
+        ...widget.options
       }
 
-      bridge.animationControls(widget.element, widget.column, options);
+      bridge.animationControls(widget.element, widget.value, options);
+      break;
+    case 'time-series':
+      bridge.timeSeries(widget.element, widget.value, widget.options);
```

Comment:
Just a question. Why are you using `widgets.options` in timeSeries and `options` in animationControls?

Reply:
I wasn't taking into account the possibility of using the `prop` parameter to change the property since we didn't have this feature in the time series widget, but now we can. I'm uploading the change and also added an example 👌

Reply:
:+1: